packages/client/src/components/remove-liquidity.tsx

```diff
@@ -297,6 +297,13 @@ function RemoveLiquidity({
             value: baseMsgValue, // flat fee sent to contract - 0.0005 ETH
         });
 
+
+        try {
+            mixpanel.track('transaction:removeLiquidity', { amount: gasEstimate });
```

Comment:
I think the amount is not correct here - `gasEstimate` is the fee. I think we should track the following (all available in the local scope):

* `pairData.id`
* `exitToken`
* `gasEstimate`
* `exitAmount`